app/src/main/java/fr/free/nrw/commons/upload/depicts/DepictsFragment.java

```diff
@@ -149,7 +153,7 @@ public void setDepictsList(List<DepictedItem> depictedItemList) {
 
     @OnClick(R.id.depicts_next)
     public void onNextButtonClicked() {
-        presenter.verifyDepictions();
+        presenter.verifyDepictions(getActivity().getApplication());
```

Comment:
Why are we passing the context to the presenter? Let it be the other way around, the presenter should ask the view for the things which need context